clang/test/Driver/gcc-prefix.cpp

```diff
@@ -0,0 +1,8 @@
+// UNSUPPORTED: system-windows
+//
```

Comment:
Drop `^//$` lines. The conventional style doesn't add these markers for otherwise blank lines. Without them, we can navigate among the tests using vim-style `{` `}` .